solidity/test/TestTokenStake.js

```diff
@@ -61,6 +63,25 @@ contract('TokenStaking', function(accounts) {
     );
   }
 
+  it("should get correct minimum stake based on the minimum stake schedule", async () => {
```

Comment:
Instead of updating the minimum stake schedule, can we go with the original values and split this single test into multiple `it`s where we'll cover all corner cases?
```
describe("minimumStake()", async () => {
  it("returns 100k when the schedule starts", ...
  it("returns 100k just before the first step ends", ...
  it("returns 50k when the second step starts"...
  ...
}
```

We don't need to cover all steps but having the first two, the last two and something in between could be helpful.